src/game.cpp

```diff
@@ -3853,25 +3851,39 @@ void Game::updateFrame(std::vector<aabb3f> &highlight_boxes,
 		Inventory
 	*/
 
-	if (client->getPlayerItem() != runData->new_playeritem)
+	bool inventory_selection_changed = false;
+	bool inventory_changed = false;
```

Comment:
It's better to be sure that the variable is initialized correctly than reporting to standard which could not be correctly followed by some non-free OSes :)